packages/driver/cypress/integration/dom/elements_spec.ts

```diff
@@ -135,38 +135,43 @@ describe('src/dom/elements', () => {
     })
 
     it('DOM element does not exist', () => {
-      cy.visit('/fixtures/dom.html')
-      const $el = $('foo-bar-baz')
+      cy.visit('/fixtures/dom.html').then(() => {
```

Comment:
Were these tests previously passing because `dom.html` was already loaded? It might not be necessary to re-visit in these tests at all.

Reply:
We need to visit in each test with session support enables since we visit 'blank' between each test.